web/client/utils/LayersUtils.js

```diff
@@ -9,21 +9,41 @@
 const assign = require('object-assign');
 const {isObject, isArray} = require('lodash');
 
+const getGroup = (groupId, groups) => {
+    return groups.filter((subGroup) => isObject(subGroup) && subGroup.id === groupId).pop();
```

Comment:
The id is autogenerated and is the same then before, are name and title that change.